src/main/java/lotto/model/Lottoes.java

```diff
@@ -24,8 +25,23 @@ public int size() {
         return lottoes.size();
     }
 
+    public Lottoes combine(Lottoes other) {
+        List<Lotto> result = new ArrayList<>();
+        result.addAll(lottoes);
+        result.addAll(other.lottoes);
+        return new Lottoes(result);
+    }
```

Comment:
개인적으로 result같은 가변객체를 지양하고싶어서
```suggestion
    public Lottoes combine(Lottoes other) {
        return Stream.concat(lottoes.stream(), other.lottoes.stream())
            .collect(collectingAndThen(toList(), Lottoes::new));
    }
```

이렇게 작성하는 편입니다 🙂

Reply:
오 Stream에 저런 기능도 있군요 ㅎㅎㅎ
감사합니다~~